<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task3</class>
 <widget class="QMainWindow" name="Task3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="addRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>add after last</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="removeRadioButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>remove first</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QComboBox" name="listComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>List1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>List2</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="listElemLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>171</y>
      <width>120</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="interactPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>remove/add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="List1ToList2pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>copy List1-&gt;List2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="List2ToList1pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>copy List2-&gt;List1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fillListPushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>random 5 elems</string>
    </property>
   </widget>
   <widget class="QLabel" name="list1Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>List1 = </string>
    </property>
   </widget>
   <widget class="QLabel" name="list2Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>List2 =</string>
    </property>
   </widget>
   <zorder>addRadioButton</zorder>
   <zorder>removeRadioButton</zorder>
   <zorder>listElemLineEdit</zorder>
   <zorder>interactPushButton</zorder>
   <zorder>listComboBox</zorder>
   <zorder>List1ToList2pushButton</zorder>
   <zorder>List2ToList1pushButton</zorder>
   <zorder>fillListPushButton</zorder>
   <zorder>list1Label</zorder>
   <zorder>list2Label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
